spec/ruby/core/complex/inspect_spec.rb

```diff
@@ -17,18 +17,18 @@
 
   it "calls #inspect on real and imaginary" do
     real = NumericSpecs::Subclass.new
-    real.should_receive(:inspect).and_return("1")
+    real.should_receive(:inspect).and_return(+"1")
```

Comment:
It feels unexpected this String is mutated, would you know where it's mutated?

Reply:
Sorry, I have thousands of these to update so I'm not digging too much in every single one.